### ISSUE TITLE
When should I set --Freq_fixed=TRUE in calculating inbreeding Coefficient?

### ISSUE THREAD

**Issue by wul88** [fgvieira/ngsF#21]
Hi, Filipe,

I am using ngsF to estimate the inbreeding coefficient for a population of giraffes.  I am not sure about the parameter "--freq_fixed". The readMe stated "assume initial MAF as fixed parameters (only estimates F)".  However, the examples shown in the ngsTools tutorials didn't mention this parameter in either initial values estimate (with --approx_EM) or the final run.

I tried running ngsF with and without setting "--freq_fixed=TRUE". The results are quite different. I would like to learn more about this parameter.

**Comment by fgvieira** [comment]
This option is only useful in few cases, when you don't want to estimate allele frequencies from your data, usually when you already have the freqs from another set of individuals of the same population.

**Comment by wul88** [comment]
Thank you very much for responding quickly!!

We try to assess the inbreeding situation of this population of giraffes. So it is important we understand the method very well. I went through the original (2013) paper. The allele frequency and inbreeding coefficient per individual per site are both estimated in each iteration. If using "--freq_fixed=TRUE", does that mean the allele frequency per site is fixed and taken  out of the iteration updating? 

I also found the ngsF downloaded directly from https://github.com/fgvieira/ngsF (Dec 19 2022 @ 17:07:30) and the one from ngsTools (Jan  4 2023 @ 20:27:03) produced different estimation of inbreeding coefficient from the same set of BEST of initail_values. Although they both are version 1.2.0-STD but different date in the brackets (see above brackets) following the version number. Which one should I choose? I have 100 low coverage (average peak depth of reads distribution excluding chromosomeX is 1.7~3) samples. The genome size is 2.5G.
Thank you again!

**Comment by fgvieira** [comment]
Yep! With `--freq_fixed=TRUE` per site freqs are fixed, and only per individual inbreeding coef are estimated.
